lmnet/executor/train.py

```diff
@@ -40,7 +40,12 @@ def setup_dataset(config, subset, rank):
     dataset_kwargs = dict((key.lower(), val) for key, val in config.DATASET.items())
     dataset = DatasetClass(subset=subset, **dataset_kwargs)
     enable_prefetch = dataset_kwargs.pop("enable_prefetch", False)
-    return DatasetIterator(dataset, seed=rank, enable_prefetch=enable_prefetch)
+    batch_pre_fetch = config.DATASET.BATCH_PRE_FETCH
```

Comment:
Maybe, the line raise error, if config don't have `DATASET.BATCH_PRE_FETCH`. Is it tested other config file?

Reply:
@ruimashita San:
I got your point. I thought all the config variables are mandatory. Anyway, I've handled it by providing default values if unavailable. Thank you for pointing out.